cakefuzzer/phpfiles/instrumentation/install_php_parser.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Download the PHP-Parser library
+curl -L -o php-parser.zip https://github.com/nikic/PHP-Parser/archive/refs/heads/master.zip
```

Comment:
```suggestion
curl -Lso php-parser.zip https://github.com/nikic/PHP-Parser/archive/refs/heads/master.zip
```
To limit verbose mode.